modules/api/php/endpoints/candidate/visit/image/format.class.inc

```diff
@@ -90,7 +90,12 @@ class Format extends Endpoint
             $handler = new Format\Brainbrowser($this->_image);
             break;
         case 'thumbnail':
-            $handler = new Format\Thumbnail($this->_image);
+        case 'nifti':
+        case 'bval':
+        case 'bvec':
+        case 'bidsjson':
+            error_log(print_r($format, true));
```

Comment:
this can be removed

Reply:
ooops probably what makes phan fail, lol